types/src/core/index.d.ts

```diff
@@ -1,15 +1,15 @@
 import { Channel } from './channels';
-export type InstallId = string;
-export type UserId = string;
-type Jsonish = string | number | boolean | null | {
+export declare type InstallId = string;
```

Comment:
Not 100% sure why these changes to declarations came in, presume from the TS compiler version or something.

I ran an `npm ci` so got what's in the lock file.